### DISCUSSION TITLE
Displaying WalletConnect QR Code by Default with AppKi

### DISCUSSION THREAD

**Discussion by bolom** [reown-com/appkit#3336] [General]
I would like to know if there is a way with AppKit to display only the WalletConnect QR code by default.
I'm building an app where user A will need user B to sign with their wallet to interact with the app.

**Comment by bolom** [comment]
basically there is way to display only walletconnect ?

**Comment by bolom** [comment]
`modal.open({ view: 'ConnectingWalletConnect' })`

this is the solution

**Comment by arein** [comment]
Glad you found it @bolom - welcome to make a Pull Request to our docs if you think you'd discover this easier somehow then
